### DISCUSSION TITLE
API Freeradius

### DISCUSSION THREAD

**Discussion by philippenc** [jaredhendrickson13/pfsense-api#425] [Ideas]
hello, are you going to create an api to create users for freeradius?

**Comment by jaredhendrickson13** [comment]
There is an open feature request (#21) for this, but there's been no movement on this from either myself or any contributors that I'm aware of. It's not actively planned but could be picked up in a future release, likely as a v2 endpoint. Feel free to follow that issue for any updates.

Thanks!
